libs/watt/package/data/watt-data-table.component.ts

```diff
@@ -152,6 +154,7 @@ export class WattDataTableComponent {
   enableSearch = input(true);
   enableRetry = input(false);
   enableCount = input(true);
+  enableEmptyState = input(true);
```

Comment:
Question: What is the reason for not showing the empty state? Asking because right now its the default on all other tables.

Reply:
We are in a sort of weird state where the page is designed as a table but really is not yet. So this is done to be able to follow the UX design and making it ready for a table. Try taking a look in figma at "hullerlog"